schemas/2019-08-01-preview/Microsoft.OperationalInsights.json

```diff
@@ -91,9 +101,16 @@
       "type": "object",
       "properties": {
         "type": {
-          "type": "string",
-          "enum": [
-            "SystemAssigned","None"
+          "oneOf": [
+            {
+              "type": "string",
+              "enum": [
+                "SystemAssigned"
```

Comment:
MSI RP requires support for both SystemAssigned and None identities. Here's 'None' is missing. Should we change API to generate both?

Reply:
Yes if you want to declare support for "None", you'll need to [update your swagger spec](https://github.com/Azure/azure-rest-api-specs/blob/454b15b1a325e019606c16d96350440f7abd6d0c/specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2019-08-01-preview/Clusters.json#L489) - that way all consumers of your API (SDKs, PowerShell, CLIs, schemas once you've onboarded) will know about it.

By the way, the [common set of definitions](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json#L216) 
(shared between multiple RPs) doesn't mention "None" - is that just an omission?

Reply:
Thanks, we'll update swagger as well